docs/source/contributing.md

```diff
@@ -1,12 +1,46 @@
 # Contributing
 
-This document is a work progress. It should describe how to contribute (code or issues) to the MapSwipe back-end.
 
-To contribute to the MapSwipe back-end please create dedicated feature branches.
+## Clone from GitHub
+
+```bash
+git clone https://github.com/mapswipe/python-mapswipe-workers.git
+cd python-mapswipe-workers
+git checkout dev
+```
+
+
+## Install MapSwipe Workers
+
+Create a Python virtual environment and activate it. Install MapSwipe Workers using pip:
+
+```bash
+python -m venv venv
+source venv/bin/activate
+pip install --editable .
+```
+
+
+## Feature Branch
+
+To contribute to the MapSwipe back-end please create dedicated feature branches:
 
 ```bash
 git checkout -b featureA
 git commit -am 'add new project type'
 git push -u origin featureA
 git request-pull origin/master featureA
```

Comment:
I think in this line it should be `git request-pull origin/dev featureA`, right? `dev` instead `master`

Reply:
Added prefered branch to make a pull request and added note about hotfix branch.